apps/crm/src/pages/_app.tsx

```diff
@@ -0,0 +1,57 @@
+import "@saleor/macaw-ui/next/style";
+import "../styles/globals.css";
+
+import { AppBridge, AppBridgeProvider } from "@saleor/app-sdk/app-bridge";
+import { RoutePropagator } from "@saleor/app-sdk/app-bridge/next";
+import React, { ReactElement, ReactNode } from "react";
+import { AppProps } from "next/app";
+
+import { NoSSRWrapper } from "@saleor/apps-shared";
+import { trpcClient } from "../modules/trpc/trpc-client";
+import { Box, ThemeProvider } from "@saleor/macaw-ui/next";
+
+import { NextPage } from "next";
+import GraphQLProvider from "../lib/graphql-provider";
+import { AppBridgeStorageSetter } from "../lib/app-bridge-persistence";
+
+/**
+ * Ensure instance is a singleton.
+ * TODO: This is React 18 issue, consider hiding this workaround inside app-sdk
+ */
+export let appBridgeInstance: AppBridge | undefined;
+
+if (typeof window !== "undefined" && !appBridgeInstance) {
+  appBridgeInstance = new AppBridge();
+}
+
+export type NextPageWithLayoutOverwrite<P = {}, IP = P> = NextPage<P, IP> & {
```

Comment:
We could use a comment about why it is needed. Is `getLayout` not sufficient?

Reply:
This is implementation of `getLayout` which is not built-in, its a pattern. I change the name, because by default layout is set, so its actually overwriting. `getLayout` suggests that if its not set, there will be "no layout"

I can add comment